apps/site/lib/site_web/controllers/cms_controller.ex

```diff
@@ -43,6 +43,22 @@ defmodule SiteWeb.CMSController do
     |> handle_page_response(conn)
   end
 
+  def reset_cache_key(conn, %{"object" => object, "id" => id}) do
+    CMS.RedisRepo.delete("/cms/#{object}/#{id}")
+
+    Logger.info("cms.redisrepo.delete path=#{object}/#{id}")
```

Comment:
We'll know that this endpoint is being hit by Drupal once this PR and that PR are promoted.